conversational-memory/src/main/java/org/opensearch/ml/conversational/action/memory/conversation/CreateConversationAction.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Aryn, Inc 2023
```

Comment:
Thanks for contributing. Are you going to keep copyright as Aryn?

Reply:
just changed em to 
/*
 * Copyright 2023 Aryn
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

Reply:
@ylwu-amzn I'm not sure what the right thing to do is here, since I'm not sure I can call myself an OpenSearch contributor just yet. This was @mashah 's recommendation

Reply:
@HenryL27 , I think you definitely are OpenSearch contributor. I will wait to check if any other guys from community has concerns or not. 

I know @reta contributed a lot to OpenSearch. Do you have any concern for adding `Copyright 2023 Aryn` from community perspective?

Reply:
@HenryL27 you are certainly contributor (DCO) which is covered by this generic statement: 

```
 * Copyright OpenSearch Contributors
```

We do not include individual contributors (individuals or organizations) afaik

Reply:
@austintlee Agree, we can go ahead and review code first. @jngz-es is reviewing now. I will also review the code.

Reply:
Hi there, +1 from the AWS Open Source Program Office - many explicit copyright statements in each file incur a lot of tech debt. 

OpenSearch is following the style of other projects, like the Apache Software Foundation's, to summarize copyrights belonging to the project contributors rather than list them in each source file. Source control provides the information on who has touched what should that become required. Happy to discuss with the Aryn open source leaders/program office if needed.

Reply:
Henri --

There are two scenarios to consider:

1. Company X has a feature or capability that they developed internally and would like to contribute it back to the OpenSearch community.

2. Contributors from company X are updating existing files, incrementally improving existing features, or collaborating with multiple contributors across organizations on new features.

In scenario #1, we believe it's beneficial to keep the lineage of the work. It incentivizes companies to contribute back to the community that which they've already invested in and solely own. Lineage of contribution is kept intact and easily distinguished.

In scenario #2, we agree listing all the companies and their copyright would be messy.

Our opinion is that for conversational memory, we are in scenario #1. I also believe that we've followed this model with Elastic and Open Source Connections and acknowledged contributions as such in OpenSearch.

Reply:
Also,

I'm not sure that I understand the "tech debt" statement. Copyright statements are not code and incur no overhead.